code/modules/antagonists/ert/ert.dm

```diff
@@ -168,6 +168,8 @@
 /datum/antagonist/ert/intern/leader/unarmed
 	outfit = /datum/outfit/centcom/centcom_intern/leader/unarmed
 
+//SKYRAT EDIT REMOVAL BEGIN
+/*
 /datum/antagonist/ert/clown
```

Comment:
again, admin spawn, and clowns exist, if you're "nuking lrp", you should remove the clown job and the pizza ert

Reply:
_there's a pizza ert?_

Reply:
yes